### ISSUE TITLE
Problem about labels

### ISSUE THREAD

**Issue by Camouflage022** [cyvius96/prototypical-network-pytorch#18]
label = torch.arange(args.train_way).repeat(args.query)
why labels is not from train_loader? It wont affect accuracy?
